src/components/Home/movie-row/MovieRow.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Movie from "./movie/Movie";
+
+interface Props {
+  genere_data: { id: number; name: string; movies: any[] };
```

Comment:
Stick with one naming convention in JS use only camelCase and in python snake_case